accessible/tree.service.js

```diff
@@ -484,8 +484,8 @@ blocklyApp.TreeService = ng.core.Class({
         // Return the focus to the workspace tree containing the input field.
         document.getElementById(treeId).focus();
 
-        // Note that Tab events are allowed to propagate through.
-        if (e.keyCode == 27) {
+        // Note that Tab and Enter events are allowed to propagate through.
```

Comment:
Fair question. It does stop propagation, so I've fixed the comment to clarify what's happening here - the behavior from those are handled on a different listener, so the propagation of this event needs to stop here.